components/Settings/FavoritesTab.js

```diff
@@ -0,0 +1,25 @@
+import ClubTableRow from '../ClubTableRow'
+
+export default (props) => {
+  const { updateFavorites, favorites, clubs } = props
+
+  const findClub = clubs ? code => clubs.find(club => club.code === code) : {}
```

Comment:
```suggestion
  const findClub = clubs ? code => clubs.find(club => club.code === code) : () => {}
```
Shouldn't it be a function in both cases?